go/roothash/api/commitment/pool.go

```diff
@@ -661,143 +502,3 @@ func (p *Pool) GetExecutorCommitments() (result []ExecutorCommitment) {
-//
-// Note that any signatures being invalid will result in no changes to the pool.
-func (m *MultiPool) addExecutorCommitments(
-	ctx context.Context,
-	blk *block.Block,
-	sv SignatureVerifier,
-	nl NodeLookup,
-	commitments []ExecutorCommitment,
-) error {
-	// Batch verify all of the signatures at once.
-	msgs := make([][]byte, 0, len(commitments))
-	sigs := make([]signature.Signature, 0, len(commitments))
-	for i := range commitments {
-		v := commitments[i] // This is deliberate.
-		msgs = append(msgs, v.Blob)
-		sigs = append(sigs, v.Signature)
-	}
-
-	if !signature.VerifyBatch(ExecutorSignatureContext, msgs, sigs) {
-		return signature.ErrVerifyFailed
-	}
-
-	// Ok, all of the signatures are valid, deserialize the blobs and add them
-	// serially.
-	var hadError bool
-	for _, v := range commitments {
-		var body ComputeBody
-		if err := cbor.Unmarshal(v.Blob, &body); err != nil {
-			hadError = true
-			continue
-		}
-
-		openCom := &OpenExecutorCommitment{
-			ExecutorCommitment: v,
-			Body:               &body,
-		}
-
-		p := m.Committees[openCom.Body.CommitteeID]
-		if p == nil {
-			hadError = true
-			continue
-		}
-
-		err := p.addOpenExecutorCommitment(ctx, blk, sv, nl, openCom)
-		switch err {
-		case nil, ErrAlreadyCommitted:
-		default:
-			hadError = true
-		}
-	}
-	if hadError {
-		return ErrBadExecutorCommits
-	}
-
-	return nil
-}
-
-// CheckEnoughCommitments checks if there are enough commitments in the pool to be
-// able to perform discrepancy detection.
-//
-// Note that this checks all committees in the multi-pool and returns an error if
-// any doesn't have enoguh commitments.
-func (m *MultiPool) CheckEnoughCommitments() error {
-	for _, p := range m.Committees {
-		if err := p.CheckEnoughCommitments(false); err != nil {
-			return err
-		}
-	}
-	return nil
-}
-
-// GetExecutorCommitments returns a list of executor commitments in the pool.
-func (m *MultiPool) GetExecutorCommitments() (result []ExecutorCommitment) {
-	for _, p := range m.Committees {
-		for _, c := range p.ExecuteCommitments {
-			result = append(result, c.ExecutorCommitment)
-		}
-	}
-	return
-}
-
-// GetTimeoutCommittees returns a list of committee pools that are up for their
-// TryFinalize to be called.
-func (m *MultiPool) GetTimeoutCommittees(now time.Time) (result []*Pool) {
-	for _, p := range m.Committees {
-		if p.IsTimeout(now) {
-			result = append(result, p)
-		}
-	}
-	return
-}
-
-// GetNextTimeout returns the minimum next timeout of all committee pools.
-func (m *MultiPool) GetNextTimeout() (timeout time.Time) {
-	for _, p := range m.Committees {
-		if timeout.IsZero() || (!p.NextTimeout.IsZero() && p.NextTimeout.Before(timeout)) {
-			timeout = p.NextTimeout
-		}
-	}
-	return
-}
-
-// ResetCommitments resets the commitments in the pool and clears their discrepancy
-// flags.
-func (m *MultiPool) ResetCommitments() {
-	for _, p := range m.Committees {
-		p.ResetCommitments()
-	}
-}
```

Comment:
what were the changes to this file since the last push?

Reply:
The only changes were fixes addressing your comments above. GitHub should really add a feature to show diffs to what you acked before.

Reply:
thanks